src/lib.rs

```diff
@@ -100,7 +111,11 @@ pub async fn run_trin(
         beacon_event_tx,
         beacon_jsonrpc_tx,
         beacon_event_stream,
-    ) = if trin_config.networks.iter().any(|val| val == BEACON_NETWORK) {
+    ) = if trin_config
+        .portal_subnetworks
```

Comment:
Think we could also do a custom `trin_config.portal_subnetwork.contains()` or something similar